src/test/java/seedu/address/logic/commands/DelEventCommandTest.java

```diff
@@ -0,0 +1,109 @@
+import static seedu.address.logic.commands.CommandTestUtil.showEventAtIndex;
+import static seedu.address.testutil.TypicalEvents.getTypicalAddressBook;
+import static seedu.address.testutil.TypicalIndexes.INDEX_FIRST_EVENT;
+import static seedu.address.testutil.TypicalIndexes.INDEX_SECOND_EVENT;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+import seedu.address.model.event.Event;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for
+ * {@code DelEventCommand}.
+ */
+public class DelEventCommandTest {
+
+    private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
+
+    @Test
+    public void execute_validIndexUnfilteredList_success() {
+        Event eventToDelete = model.getFilteredEventList().get(INDEX_FIRST_EVENT.getZeroBased());
+        DelEventCommand deleventCommand = new DelEventCommand(INDEX_FIRST_EVENT);
+
+        String expectedMessage = String.format(DelEventCommand.MESSAGE_DELETE_EVENT_SUCCESS, eventToDelete);
+
+        ModelManager expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+        expectedModel.deleteEvent(eventToDelete);
+
+        assertCommandSuccess(deleventCommand, model, expectedMessage, expectedModel);
+    }
+
+    @Test
+    public void execute_invalidIndexUnfilteredList_throwsCommandException() {
+        Index outOfBoundIndex = Index.fromOneBased(model.getFilteredEventList().size() + 1);
+        DelEventCommand deleventCommand = new DelEventCommand(outOfBoundIndex);
+
+        assertCommandFailure(deleventCommand, model, Messages.MESSAGE_INVALID_EVENT_DISPLAYED_INDEX);
+    }
+
+    @Test
+    public void execute_validIndexFilteredList_success() {
+        showEventAtIndex(model, INDEX_FIRST_EVENT);
+
+        Event eventToDelete = model.getFilteredEventList().get(INDEX_FIRST_EVENT.getZeroBased());
+        DelEventCommand deleventCommand = new DelEventCommand(INDEX_FIRST_EVENT);
+
+        String expectedMessage = String.format(DelEventCommand.MESSAGE_DELETE_EVENT_SUCCESS, eventToDelete);
+
+        Model expectedModel = new ModelManager(model.getAddressBook(), new UserPrefs());
+        expectedModel.deleteEvent(eventToDelete);
+        showNoEvent(expectedModel);
+
+        assertCommandSuccess(deleventCommand, model, expectedMessage, expectedModel);
+    }
+
+    @Test
+    public void execute_invalidIndexFilteredList_throwsCommandException() {
+        showEventAtIndex(model, INDEX_FIRST_EVENT);
+
+        Index outOfBoundIndex = INDEX_SECOND_EVENT;
+        // ensures that outOfBoundIndex is still in bounds of address book list
+        assertTrue(outOfBoundIndex.getZeroBased() < model.getAddressBook().getEventList().size());
+
+        DelEventCommand deleventCommand = new DelEventCommand(outOfBoundIndex);
```

Comment:
Would it be better to capitalise the letter E? I think the letter E in other delevent variables are capitalised too.

```suggestion
        DelEventCommand delEventCommand = new DelEventCommand(outOfBoundIndex);
```

Reply:
Edited, thanks!